src/Tests.IntegrationTests/ClassifyMessageTests.cs

```diff
@@ -31,24 +34,29 @@ public async Task EnvironmentIsConfigured()
 
             // Assert
             _ = Assert.Single(institutions);
-            Assert.Equal(2, participants.Count);
+            Assert.Equal(3, participants.Count);
             Assert.Equal(participants[0].InstitutionId, institutions.Single().Id);
             Assert.Equal(participants[1].InstitutionId, institutions.Single().Id);
         }
 
         [Fact(Timeout = 120000)]
         public async Task GivenValidMessageReceivesValidResponse()
         {
+            _output.WriteLine("Starting GivenValidMessageReceivesValidResponse");
```

Comment:
I'm not sure if any of these output lines are sticking around? I'll provide suggestions if they are going to be remain in the code.

Use nameof:
```suggestion
            _output.WriteLine($"Starting {nameof(GivenValidMessageReceivesValidResponse)}");
```

Reply:
yes they are sticking around for now at least. They show up in the Actions log so useful to see